### DISCUSSION TITLE
why Could not find an active window.

### DISCUSSION THREAD

**Discussion by superen7527** [floccusaddon/floccus#1518] [Q&A] [answered]
when i use it in kiwi(Android browser)

i cant add any url. always hint "Could not find an active window."

please help me,thx

**Comment by 3mile** [comment]
same question

**Comment by marcelklehr** [comment]
please open a new issue with all details of your setup

**Comment by nam2009** [comment]
The latest version cannot solve this problem. Please fix this problem as soon as possible.

**Reply by nam2009** [reply]
WEBdav url always hint "Could not find an active window."(kiwi__Android browser)

**Comment by marcelklehr** [comment, marked as answer]
See https://github.com/floccusaddon/floccus/issues/1564
